contracts/StablecoinConverter.sol

```diff
@@ -395,6 +397,14 @@ contract StablecoinConverter is EpochTokenLocker {
         addBalanceAndBlockWithdrawForThisBatch(msg.sender, tokenIdToAddressMap(0), feeReward);
     }
 
+    /** @dev called during solution submission to burn fees from previous auction
+      */
+    function burnPreviousAuctionFees() internal {
+        if (!currentBatchHasSolution()) {
+            feeToken.burnOWL(address(this), latestSolution.feeReward);
```

Comment:
Interesting, you have to approve to burn your own OWL?

Reply:
Super weird...